File: chap3/julia/code6.jl
using Flux

z1 = Dense(784, 256)
batchNorm = BatchNorm(256, relu)
z2 = Dense(256, 128, relu)
y = Dense(128, 10)
model = Chain(
  z1, 
  batchNorm,
  z2, 
  y, 
  softmax
)

File: chap3/R/code6.r
x = layer_input(784)
z1 = x %>% layer_dense(units = 256, activation="relu")
batch_norm = z1 %>% layer_batch_normalization()
z2 = batch_norm %>% layer_dense(units = 128, activation="relu")
y = z2 %>% layer_dense(units = 10, activation="softmax")
model = keras_model(x, y)
model